frontend/src/lib/components/Support/SupportModal.tsx

```diff
@@ -40,6 +40,31 @@ const SUPPORT_TICKET_KIND_TO_PROMPT: Record<SupportTicketKind, string> = {
     feedback: 'What feedback do you have?',
     support: 'What can we help you with?',
 }
+const SUPPORT_TICKET_KIND_TO_PROMPT_EXTENDED_1: Record<SupportTicketKind, string> = {
+    bug: "What's the bug?",
+    feedback: 'Is your feedback related to a problem? (make sure to include links if relevant)',
+    support: 'What can we help you with? (make sure to include links if relevant)',
+}
+const SUPPORT_TICKET_KIND_TO_PROMPT_EXTENDED_2: Record<SupportTicketKind, string> = {
+    bug: 'How to reproduce?',
+    feedback: "Describe the solution you'd like?",
+    support: 'What is happening?',
+}
+const SUPPORT_TICKET_KIND_TO_PROMPT_EXTENDED_3: Record<SupportTicketKind, string> = {
+    bug: 'Additional context? (make sure to include links if relevant)',
+    feedback: "Describe alternatives you've considered.",
+    support: 'What do you expect to happen?',
+}
+
+// topics that will ask for additional information
+export const EXTENDED_SUPPORT_TOPICS: string[] = [
+    'app_performance',
+    'analytics',
+    'data_management',
+    'apps',
+    'data_integrity',
+    'ingestion',
+]
```

Comment:
let's not do this, we're definitely going to forget adding items here and it doesn't really make sense to users either

Reply:
nothing is topic specific -> it's the support kind that determines what we ask, if you want something specific you can add a special support kind for it.